game.py

```diff
@@ -10,8 +13,9 @@ def run_game():
         print(f"Добре да се забавляваме, {username}. Нека да започнем играта!")
         incorrect_count = 0
         correct_count = 0
-
+        total_answer = 0
         for i in range(1, 11):
+            total_answer += i
```

Comment:
It is very interesting that we are keeping track of the total answer count now, let's also export this in the CSV file!